theano/gpuarray/subtensor.py

```diff
@@ -1106,15 +1106,19 @@ def gpu_kernels(self, node, nodename):
   old = *base;
   do {
     assumed = old;
-    sum = __float2half_rn(
+    ga_half old_perm;
+    __HALF_TO_US(old_perm)  = __byte_perm(old, 0,
```

Comment:
Well, tests that use atomics, fail (compile errors) if I don't make this change.